datadog/resource_datadog_dashboard.go

```diff
@@ -3811,6 +3811,8 @@ func getTimeseriesRequestSchema() map[string]*schema.Schema {
 		"q":             getMetricQuerySchema(),
 		"apm_query":     getApmOrLogQuerySchema(),
 		"log_query":     getApmOrLogQuerySchema(),
+		"rum_query":     getApmOrLogQuerySchema(),
+		"network_query": getApmOrLogQuerySchema(),
```

Comment:
@gzussa can we rename the `getApmOrLogQuery` function to reflect RUM and network too?